stratum/hal/bin/barefoot/README.md

```diff
@@ -83,14 +83,19 @@ export BSP_PATH=`pwd`/bf-reference-bsp-<SDE_VERSION>
 Replace step 3 in the sequence above with:
 ```
 cd $SDE/p4studio_build
-sed -i.bak '/package_dependencies/d; /thrift/d' profiles/stratum_profile.yaml  # For SDE version <= 8.9.1
+sed -i.bak '/package_dependencies/d; /thrift/d' profiles/stratum_profile.yaml  # For SDE version <= 8.9.x
 ./p4studio_build.py -up profiles/stratum_profile.yaml --bsp-path $BSP_PATH
 ```
 
 You may also still install the BSP manually. If you are not using the reference
 BSP, you will need to install the BSP yourself (under `$BF_SDE_INSTALL`) based
 on your vendor's instructions.
 
+### Supported SDE versions
+
+ - 8.9.2
```

Comment:
This is inconsistent with the statement above: "These instructions are valid for SDE versions 8.8.0 through 9.0.0."